### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by rdthetha**
Added  home page of the website

### CODE REVIEW

**Review by shangkaul — APPROVED** [review]
Looks, great! Ready to merge 🚀

Small nuance - Can add a search Icon to search bar (just a suggestion from my POV)

File: home.css [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    min-width: 10rem;
+}
+
+.small-text {
+
+    margin-top: 1rem;
+}
+
+.product-section {
+
```
Comment (sri-rishi) [inline home.css:141]: remove all extra spacing in files, Otherwise looks good to merge

**Review by sri-rishi — CHANGES_REQUESTED** [review]

File: home.css (hunk 1 above)
Comment (rdthetha) [reply home.css:141]: Sure! would remove it!

**Review by AshutoshM29 — COMMENTED** [review]
Great work mate!
you can merge it 🚀

**Pull request merged**